### DISCUSSION TITLE
Review input data

### DISCUSSION THREAD

**Discussion by GuillaumeVandekerckhove** [DOV-Vlaanderen/pygada#7] [Review]
@Roel, is it possible to review;

- [ ] [groundwater data pydov](https://github.com/DOV-Vlaanderen/pygada/blob/main/pygada/data_input/groundwater_pydov.py)

- [ ] [soil data pydov](https://github.com/DOV-Vlaanderen/pygada/blob/main/pygada/data_input/soil_pydov.py)

**Comment by Roel** [comment]
I took a look at both, and have a few comments, questions and ideas. Maybe it will be easiest to go through them together?

**Reply by GuillaumeVandekerckhove** [reply]
Thanks for the review!

**Reply by Roel** [reply]
I put my ideas in a draft PR so you can take a look already if you want: https://github.com/DOV-Vlaanderen/pygada/pull/8
